backend/app/controllers/api/v1/passwords_controller.rb

```diff
@@ -1,24 +1,19 @@
 class Api::V1::PasswordsController < DeviseTokenAuth::PasswordsController
-  before_action :set_user
+  before_action :check_login
 
-  def update    
-    if @user.update_with_password(password_edit_params)
-      render json: { user: @user }, status: 200
-    else
-      render json: { errors: @user.errors.full_messages }, status: 422
-    end
+  def update
+    current_api_v1_user.update_with_password(password_edit_params)
+
+    render json: { errors: current_api_v1_user.errors.full_messages }, status: 422 and return if bookmark.invalid?
+
+    current_api_v1_user.save!
+    render json: { user: current_api_v1_user }, status: 200
+  rescue StandardError => e
+    render json: { errors: e.message }, status: 500
```

Comment:
Devise gemのコードを読む感じupdate_with_passwordは更新に失敗したらfalse成功したらtrueを返すようになっており、例外を返す実装にはなってなさそうです(メソッド名に！がついてないことからも例外は発生しないとわかります)
そのためするなら以下のような実装になるかと思います。
またここには関係なさそうなbookmarもいるのでそこも含めていま一度実装を確認していただけると幸いです！
```suggestion
    unless current_api_v1_user.update_with_password(password_edit_params)
       render json: { errors: current_api_v1_user.errors.full_messages }, status: 422 and return
    end
      
    render json: { user: current_api_v1_user }, status: 200
```

Reply:
例外処理は削除して、booleanで判定するように修正しました